### ISSUE TITLE
custom ggplot2 theme

### ISSUE THREAD

**Issue by wjakethompson** [atlas-aai/ratlas#5]
I would like to have a custom atlas ggplot2 theme that we can add to all our graphics with something like 

`+ theme_atlas()`

My ideal look would be similar to `theme_ipsum()` from [**hrbrthemes**](https://github.com/hrbrmstr/hrbrthemes), so that's probably a good place to start in terms of how to create a theme. He also has examples of how to write tests for these functions, so we will want to mimic that as well. We'll want to make some modifications to this. For example, in `theme_ipsum` axis title are not centered by default, and have a smaller font and the axis text. It would also be good to have the legend be at the bottom of the plot by default.

In addition, I'm hoping you can google around and figure out how to set a default color palette. I ideally, we would use `palette_OkabeIto` from the [**colorblindr**](https://github.com/clauswilke/colorblindr) package for discrete scales, and `viridis` from [**viridisLite**](https://github.com/sjmgarnier/viridisLite) for continuous scales. I feel like I've seen somewhere how to change the default color scales, but I can't remember now.

What I'm hoping for is that this:

```r
library(tidyverse)
library(ratlas)

ggplot(mtcars, aes(x = mpg, y = disp)) +
  geom_point(aes(color = factor(cyl))) +
  theme_atlas()
```

Will do the same thing as:

``` r
library(tidyverse)
library(hrbrthemes)
library(colorblindr)

ggplot(mtcars, aes(x = mpg, y = disp)) +
  geom_point(aes(color = factor(cyl))) +
  scale_color_OkabeIto() +
  theme_ipsum(axis_title_just = "c",
              axis_title_size = 12,
              axis_text_size = 10) +
  theme(legend.position = "bottom")
```

![](https://i.imgur.com/6idHtTV.png)

<sup>Created on 2019-05-15 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by wjakethompson** [comment]
Another good resource may be the [**bbplot**](https://github.com/bbc/bbplot) package from BBC. They also have a [graphics cookbook](https://bbc.github.io/rcookbook/) that has some descriptions at the top about how the functions work.

**Comment by wjakethompson** [comment]
Closed by #6
